x-pack/plugins/apm/public/components/app/ServiceMap/__stories__/Cytoscape.stories.tsx

```diff
@@ -330,7 +328,7 @@ storiesOf('app/ServiceMap/Cytoscape', module)
           },
         },
       ];
-      return <Cytoscape elements={elements} height={600} width={1340} />;
+      return <Cytoscape elements={elements} height={600} />;
```

Comment:
Why did we hardcode the `width` in the first place? And why is it no longer needed? (same question about `height`)

Reply:
oh, this is for storybook. Then it's not a big deal 👍

Reply:
> Why did we hardcode the width in the first place? And why is it no longer needed? (same question about height)

The width was needed to define the `boundingBox` of the previous layout. This option isn't necessary with the new layout, so i was able to remove it. `height` is still necessary to set the root div height (which is always full width).